src/main/kotlin/app/revanced/patches/youtube/video/playback/VideoPlaybackPatch.kt

```diff
@@ -47,8 +47,8 @@ import com.android.tools.smali.dexlib2.util.MethodUtil
 @Suppress("unused")
 object VideoPlaybackPatch : BaseBytecodePatch(
     name = "Video playback",
-    description = "Adds options to customize settings related to video playback," +
-            "such as default video quality and playback speed, etc.",
+    description = "Adds options to customize settings related to video playback, " +
+            "such as default video quality and playback speed.",
```

Comment:
I removed "etc" because it is already implied that "default video quality and playback speed" are just two of several examples. Additionally, "etc" is typically used after a longer list of examples.